test/test_helper.rb

```diff
@@ -1,5 +1,15 @@
 require 'simplecov'
-SimpleCov.start 'rails'
+require 'simplecov-lcov'
+
+SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
+  SimpleCov::Formatter::LcovFormatter,
+  SimpleCov::Formatter::HTMLFormatter,
+])
```

Comment:
Layout/IndentFirstArrayElement: Indent the right bracket the same as the first position after the preceding left parenthesis.

Reply:
Fixed.